hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -27,15 +33,43 @@ define([
       $q = _$q_;
       $rootScope = _$rootScope_;
       $window = _$window_;
+      AbsenceType = _AbsenceType_;
       UtilsService = _UtilsService_;
 
-      $httpBackend.whenGET(/action=getfulldetails&entity=HRJobContract/).respond({});
-      $httpBackend.whenGET(/action=getcurrentcontract&entity=HRJobContract/).respond({});
-      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond({});
+      $httpBackend.whenGET(/action=getfulldetails&entity=HRJobContract/).respond(contractMock.contractEntity);
```

Comment:
Please move all the `httpBackend ` calls to a separate function, this beforeEach is getting too big.